Public/index_app/index.js

```diff
@@ -1,7 +1,19 @@
 var indexApp = angular.module('indexApp',[]);
 
-indexApp.controller('greetingController', ['$scope', function($scope){
+indexApp.controller('greetingController', ['$scope', '$http', '$location', function($scope, $http, $location){
+	$scope.dataObject = {};
 	$scope.greeting = "hello";
+	console.log($location);
+	$http.get('http://localhost:4567/files').then(getDataSuccess, getDataError);
+
+	function getDataSuccess(responseObject) {
```

Comment:
didn't attach these functions to `$scope` because they are just request handlers, therefore we can think of them as private controller functions that shouldn't be exposed to the dom.